content/kotlin/concepts/datetime/terms/atDate/atDate.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.atDate()'
+Description: 'Combines a date and time into a LocalDateTime object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.atDate()`** method in Kotlin is a powerful tool that combines a date and time into a `LocalDateTime` object. This method is particularly useful when you need to create a complete timestamp by merging separate `LocalDate` and `LocalTime` instances.
```

Comment:
```suggestion
The **`.atDate()`** method in Kotlin is a function that combines date and time into a `LocalDateTime` object. This method is beneficial when creating a complete timestamp by merging separate `LocalDate` and `LocalTime` instances.
```
Avoid using first- and second-person pronouns (e.g. "I", "we", "you") if possible.